assets/queries/openAPI/responses_wrong_http_status_code/query.rego

```diff
@@ -6,6 +6,7 @@ CxPolicy[result] {
 	doc := input.document[i]
 	openapi_lib.check_openapi(doc) != "undefined"
 	response := doc.paths[n][oper].responses[code]
+	openapi_lib.content_allowed(oper, code)
```

Comment:
There is a reason to check if content is allowed here? Since it is not checking content, I don't see why we should check if content is allowed...

Reply:
No content should be allowed for HEAD responses